### ISSUE TITLE
Redirects not working, version 2.0.0-pre.6 (or above?)

### ISSUE THREAD

**Issue by freshp86** [Polymer/polymer-bundler#479]
Fairly simple minimal example

```
/tmp/polymer_bundler_test> ls -R
.:
folder1  folder2

./folder1:
foo.html

./folder2:
downloads.js  style.css
```

foo.html contents
```html
<link rel="stylesheet" href="chrome://resources/css/style.css">
<script src="chrome://downloads/downloads.js"></script>
```

downloads.js contents
```js
console.log('Hello world');
```

style.css contents
```css
body { margin: 0px; }
```

polymer-bundler fails as follows

```
> ../node_modules/polymer-bundler/lib/bin/polymer-bundler.js --inline-css --inline-scripts --redirect "chrome://downloads/|/tmp/polymer_bundler_test/folder2/" --redirect "chrome://resources/css/|folder2/" --in-html ./folder1/foo.html --out-html foo.html
polymer-bundler is currently in alpha! Use at your own risk!
Unable to load script at chrome://downloads/downloads.js: Can't load URL: chrome://downloads/downloads.js
Unable to load stylesheet at chrome://resources/css/style.css: Can't load URL: chrome://resources/css/style.css
```
Which indicates that it tries to load the URLs before applying the redirects, or the redirects are completely ignored. At https://github.com/Polymer/polymer-bundler/blob/master/src/bundler.ts#L93-L125, I don't see any reference to `opts.redirects`.

On the other hand, Vulcanize version 1.15.4 succeeds as follows

```
> ../node_modules/vulcanize/bin/vulcanize --inline-css --inline-scripts --redirect "chrome://downloads/|/tmp/polymer_bundler_test/folder2/" --redirect "chrome://resources/css/|folder2/" ./folder1/foo.html
<html><head><meta charset="UTF-8"><style>
body {
 margin: 0px;
}

</style>
<script>console.log('Hello world');
</script>
</head><body></body></html>
```

**Comment by usergenic** [comment]
@danbeam @freshp86 how would it make you feel if I said that redirect should be implemented in terms of a phase in a polymer-build pipeline?

**Comment by freshp86** [comment]
I have not looked very closely into polymer-build itself, but my first impression is that it makes certain assumptions about a project's code structure which is unlikely that we can (or want) to meet from our Chromium codebase. Is my impression correct (or am I confusinsg polymer-build with Polymer CLI)?

Also just installing polymer-build brings in 49MB of dependencies, and will be a very tough sell. It sounds fairly large set of dependencies for the functionality we are seeking (generating an HTML bundle).

**Comment by usergenic** [comment]
I'll look into best options without forcing you to bring in polymer-build and/or polymer-cli.

**Comment by usergenic** [comment]
relevant: https://github.com/Polymer/polymer-analyzer/pull/666

**Comment by usergenic** [comment]
@freshp86 @danbeam I should soon be able to implement a `--redirect` option for `bin/polymer-bundler.js` as a composition of url loaders now available in analyzer without much work, preventing the need to use `polymer-build`.  This will probably have to wait until after I/O but it should be straightforward enough.

**Comment by freshp86** [comment]
@usergenic: Thanks. Let me know when it is ready (even if it is on branch other than master), so that I can try it within Chromium's context.

**Comment by usergenic** [comment]
@danbeam @freshp86 i'll make a quick branch for this and have something for you on monday.

**Comment by usergenic** [comment]
https://github.com/Polymer/polymer-bundler/pull/532

**Comment by usergenic** [comment]
Fixed in #532
